### PR TITLE
BXMSPROD-2211 removing prod kogito images nightlies

### PR DESCRIPTION
**Pull request by mareknovotny**
https://issues.redhat.com/browse/BXMSPROD-2211

**Thank you for submitting this pull request**

**JIRA**:

[[BXMSPROD-2211](https://examle.com)](https://issues.redhat.com/browse/BXMSPROD-2211)

**referenced Pull Requests**:

* https://github.com/kiegroup/kogito-pipelines/pull/49

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy [hunk 1]
```diff
@@ -50,8 +50,7 @@ pipeline{
         ${kogitoNightlyStage(KOGITO_NEXT_PRODUCT_VERSION, KOGITO_NEXT_PRODUCT_BRANCH, OPTAPLANNER_NEXT_PRODUCT_VERSION, NEXT_PRODUCT_VERSION, NEXT_RHBA_VERSION_PREFIX, KOGITO_NEXT_PRODUCT_CONFIG_BRANCH)}
```
Comment (rgdoliveira) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy:50]: are we removing RHPAM Kogito too, right? should this line here be removed too?

File: job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy [hunk 2]
```diff
@@ -47,11 +47,10 @@ pipeline{
     
     stages {
         ${rhbaNightlyStage(NEXT_PRODUCT_VERSION, NEXT_PRODUCT_BRANCH, NEXT_PRODUCT_CONFIG_BRANCH)}
-        ${kogitoNightlyStage(KOGITO_NEXT_PRODUCT_VERSION, KOGITO_NEXT_PRODUCT_BRANCH, OPTAPLANNER_NEXT_PRODUCT_VERSION, NEXT_PRODUCT_VERSION, NEXT_RHBA_VERSION_PREFIX, KOGITO_NEXT_PRODUCT_CONFIG_BRANCH)}
+        //{kogitoNightlyStage(KOGITO_NEXT_PRODUCT_VERSION, KOGITO_NEXT_PRODUCT_BRANCH, OPTAPLANNER_NEXT_PRODUCT_VERSION, NEXT_PRODUCT_VERSION, NEXT_RHBA_VERSION_PREFIX, KOGITO_NEXT_PRODUCT_CONFIG_BRANCH)}
     
         // blue
-        ${rhbaNightlyStage(NEXT_BLUE_PRODUCT_VERSION, NEXT_BLUE_PRODUCT_BRANCH, NEXT_BLUE_PRODUCT_CONFIG_BRANCH)}
```
Comment (jakubschwan) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy:53]: we also removing rhba blue nightly? Or only kogito parts are disabled/removed ?

**Review by jakubschwan — COMMENTED** [review]
just one comment, otherwise LGTM

File: job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy (hunk 2 above)
Comment (mareknovotny) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy:53]: just kogito, the below bamoe nightly stays ;)

File: job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy [hunk 3]
```diff
@@ -47,11 +47,10 @@ pipeline{
...
-        ${kogitoNightlyStage(KOGITO_BLUE_NEXT_PRODUCT_VERSION, KOGITO_BLUE_NEXT_PRODUCT_BRANCH, null, NEXT_BLUE_PRODUCT_VERSION, NEXT_BLUE_RHBA_VERSION_PREFIX, KOGITO_BLUE_NEXT_PRODUCT_CONFIG_BRANCH)}
+        ${rhbaNightlyStage(NEXT_BLUE_PRODUCT_VERSION, NEXT_BLUE_PRODUCT_BRANCH, NEXT_BLUE_PRODUCT_CONFIG_BRANCH)}
```
Comment (mareknovotny) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_rhpam_pipeline.groovy:53]: fyi @jakubschwan ^ this blue bamoe nightly is kept

**Comment by mareknovotny** [comment]
ok i am merging it now

**Pull request merged**